src/lib/components/LighthouseScoresContainer/index.js

```diff
@@ -6,39 +8,79 @@
 class LighthouseScoresContainer extends HTMLElement {
   constructor() {
     super();
-    this.firstUpdated = false;
+
+    this.metaElement = null;
+    this.statsElement = null;
+    this.metricsElement = null;
+    this.auditsElement = null;
+
+    store.subscribe(this.onStateChanged.bind(this));
+    this.onStateChanged();
+
+    this.onCategoryChanged = (ev) => {
+      this.auditsElement.filteringOn = ev.detail;
+    };
   }
 
-  connectedCallback() {
-    // LighthouseScoresContainer expects to find children elements which it manages. It's not a
-    // LitElement, so wire things up once it's connected, and keep track of whether it's done.
-    if (this.firstUpdated) {
+  onStateChanged() {
+    const {
+      lighthouseError,
+      lighthouseResult,
+      activeLighthouseUrl,
+    } = store.getState();
+
+    if (!lighthouseResult) {
```

Comment:
should we also check that `lighthouseError` exists?

Reply:
Well, it's null if it doesn't, which won't display anything. So it's moot?